base_station/gui/src/components/AutonJoystickReading.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 <div class = "main-content">
  <h3>Joystick Reading</h3>
-  <div class="wrap">
+  <div class="joystick-wrap">
```

Comment:
also interesting that an auton joystick change is in this PR, has this been rebased with main more recently than ik-integration?

Reply:
I think all these changes appeared when I merged https://github.com/tabiosg/mrover-workspace/pull/1 . @CameronTressler can take a look if these changes are fine or not

Reply:
About to push a commit to remove the science and auton stuff. Cannot f-ing wait for all gui changes to be in main

Reply:
Thank you... I should've looked more closely when I merged all of this stuff in the first place...

Reply:
No you're totally good, this is due to us having terrible git practices and having gui code scrambled together/not merged into main since F20

Reply:
Was not at all trying to say this was your fault